fmf_scheduler.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/python3
```

Comment:
Since this script actually proposes a new runtime, I would personally prefer to have it proposed in a different PR. I actually dislike that it's not integrated into colin's CLI interface.

Reply:
yep, this is my suggested solution, split this PR, to 2 or 3 PRs, and last of them will be this nosetests generator.

But on other hand, without this, it is harder to show benefits eg:
 - write tests just in metadata without empty python classes
 - using nosetests scheduled tests
 - filtering and selecting or test items
 - using FMF rulesets

And this PR will just split metadata outside python classes to FMF files.
But I agree and with this knowledge, that these PRs are closely connected I can split it.